### ISSUE TITLE
Duplicate enums are not generated

### ISSUE THREAD

**Issue by auliere** [mamift/LinqToXsdCore#31]
There are a couple of bugs related to enum generation.

1) When you have two anonymous enum types defined for _**attributes**_ within different complex types, the enum only generates for a single one of these comples types (see the example in the attachment)
[SimpleSchema.zip](https://github.com/mamift/LinqToXsdCore/files/6618226/SimpleSchema.zip)

2) When you have two anonymous enum types defined for _**elements**_ within different complex types, the enum is not generated at all (please see the example in the attachment)
[SimpleSchemaElement.zip](https://github.com/mamift/LinqToXsdCore/files/6618309/SimpleSchemaElement.zip)

**Comment by auliere** [comment]
For issue 1 - Looks like it is partially done on purpose:

https://github.com/mamift/LinqToXsdCore/blob/d6dbc2ff801e2fe4561ea140c197983fd6014735/XObjectsCode/Src/TypesToCodeDom.cs#L238-L240

After uncommenting these lines my SampleSchema produces a valid c# file.

**Comment by mamift** [comment]
Hello, 

Yes I'm aware of the issue; I've been trying to properly diagnose & fix the issue in the [multipleEnumFix branch](https://github.com/mamift/LinqToXsdCore/tree/multipleEnumFix), but due to time constraints haven't been able to get to the root cause.

**Comment by mamift** [comment]
> @mamift - I see that your most recent commit to master branch has fixed this issue. Do you know when you plan to release this version so that it can be used without building the tool from the sources?

Yes I do intend to do that; I'll see if I can get around to it this coming weekend.

**Comment by mamift** [comment]
@auliere New release should be publicly visible shortly: https://www.nuget.org/packages/LinqToXsdCore/3.2.1

**Comment by auliere** [comment]
@mamift  Thank you very much for the fix and a prompt release! Works nice and helps us to get the job done!

Looks like problem 2 still persists though - but I didn't encounter any problems with it for now in my real projects.
